packaging/installer/dependencies/gentoo.sh

```diff
@@ -24,12 +24,14 @@ package_tree="
   sys-apps/util-linux
   net-libs/libmnl
   dev-libs/json-c
+  dev-libs/libyaml
   dev-libs/libuv
   app-arch/lz4
   dev-libs/openssl
   virtual/libelf
   dev-lang/python
   dev-libs/libuv
+  dev-libs/libyaml
```

Comment:
You’ve got libyaml twice here.

Reply:
I do not have any additional comment, because I could compile this PR, @MrZammler when you have a time, please, remove this additional line.